ui/app/pages/settings/networks-tab/network-form/network-form.component.js

```diff
@@ -254,7 +261,7 @@ export default class NetworkForm extends PureComponent {
         {this.renderFormTextField(
           'rpcUrl',
           'rpc-url',
-          this.setStateWithValue('rpcUrl', this.validateUrl),
+          this.setStateWithValue('rpcUrl', this.validateUrl.bind(null, networksToRender)),
```

Comment:
We could do without this `bind` step by grabbing the networks to render from props directly in `validateUrl`.